Property_map/examples/Property_map/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+# Created by the script cgal_create_CMakeLists
+# This is the CMake script for compiling a set of CGAL applications.
+
```

Comment:
Is it really the default generated cmake script? I though the one with the glob was the default.

Reply:
I changed it, but my guess that there will be quite some CMakeLists.txt wirth that comment.

Reply:
Does not seem changed to me...

Reply:
changed.